src/include/gurt/debug_setup.h

```diff
@@ -75,7 +75,7 @@
 	/** All debug streams */                \
 	ACTION(DB_ALL,   all,   all,   0, arg)  \
 	/** Generic debug stream */             \
-	ACTION(DB_ANY,   any,   any,   0, arg)  \
+	ACTION(DB_MISC,  misc,  misc,  0, arg)  \
```

Comment:
I think ANY is fine.   The main issue I have with this is it forces a version bump in cart.   I think we should just document that DB_ANY is intended as a uncategorized debug stream but it still behaves like other debug streams in the sense that it has to be explicitly enabled.   If someone wants to always log, when any debug is enabled, they should use DB_ALL